.gitignore

```diff
@@ -11,4 +11,8 @@ pom.xml.asc
 /.lein-*
 *.iml
 .idea/
-.log/
\ No newline at end of file
+.log/
+copagame.iml
+.clj-kondo/com.github.seancorfield/next.jdbc/hooks/com/github/seancorfield/next_jdbc.clj
+
+.clj-kondo/com.github.seancorfield/next.jdbc/
```

Comment:
Everything that is not `.clj-kondo/.cache` can be kept. That's updated only when dependencies changes and I think it's generally ok to leave them here.

Reply:
Ahhh ok. Nice. Changed.